### ISSUE TITLE
Blackout Windows - As a user I want to know if I am launching experiments during a release, so that I don't run into issues

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#5654]
**TL;DR:** **Warn people about launching experiments during times that are less optimal**

**Why?** Helps reduce need for background knowledge about when is a good time to launch

Humans have made some gating choices when running experiments through Normandy.  As we remove the humans - we need to either remove or accept the pipeline issues or set up rules that Nimbus respects for when to gate launching. 

Most common case - should address early:

* Not launching experiments on [release launch day](https://wiki.mozilla.org/Release_Management/Calendar) and the 3 follow days.  It's not an issue for about 50% of experiments - but while the release is throttling in availability is not good for user enrollment and analysis.  It can create low enrollment or even biasing issues - which are much worse on any release where there are delays in scaling up (at least a couple of times a year). It's best to just avoid all together.

  * NOTE: the exception may be if Nimbus decides to implement hold back studies - which by definition need to go out with the new release.

Seasonal + "big bang" (like Shirley) black out windows.

* No launch windows.  If the company is doing a "big bang" type of activity - like Shirley - we may hold on all other experiments for a brief period. 
* We also like to do a "no launch" period in the last 2 weeks of the year - as too many people are away for holiday.  Existing experiments are left to run over that time - but new ones we don't launch in the days before so many people step away - as the data can be wonky (usage is atypical) and if something goes wrong there are fewer people to catch/remediate.

**Goal:** 

Have the system do some intelligent launch gating, queuing, and communicate to users - replacing what experiment coordinator and DS had done manually. 

* Ex: if the customer hits "launch" on the day of release (or maybe the day before?) - let them know to avoid launch complicating their results - their experiment will be launched x days after release.
* ex: if the customer hits launch - but it's during a company "big bang" release or 2 days before a company shutdown - they get notice that it will be queued until after and they may need to check the delay date impacts and adjust their target versions.

┆Issue is synchronized with this [Jira Epic](https://mozilla-hub.atlassian.net/browse/EXP-1137)

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Not started. Prioritizing smaller wins at the moment.
